test/test_cartesian_dmp.py

```diff
@@ -75,7 +75,7 @@ def test_compare_python_cython():
         current_y=np.array([1.0, 0.0, 0.0, 0.0]), current_yd=np.zeros([3]),
         goal_y=np.array([1.0, 0.0, 0.0, 0.0]), goal_yd=np.zeros([3]), goal_ydd=np.zeros([3]),
         start_y=np.array([1.0, 0.0, 0.0, 0.0]), start_yd=np.zeros([3]), start_ydd=np.zeros([3]),
-        goal_t=2.0, start_t=0.0, alpha_y=25.0, beta_y=6.25,
+        goal_t=2.0, start_t=0.0, alpha_y=25.0*np.ones(3), beta_y=6.25*np.ones(3),
```

Comment:
PEP8 suggests to add spaces around operators

Reply:
Addressed in [0584376](https://github.com/dfki-ric/movement_primitives/pull/59/commits/0584376262360124530b4f2e5704ca6fd314496b)